### PR TITLE
Update grafana comment

### PR DESCRIPTION
**Pull request by daxmc99**

### CODE REVIEW

File: docker-images/grafana/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ RUN ls '/generated/grafana'
 
 # Prepare final image
 # when upgrading the Grafana version, please refer to https://docs.sourcegraph.com/dev/background-information/observability/grafana#upgrading-grafana
-# DO NOT UPGRADE beyond 7.5.4 without consulting legal, Grafana > 7.5.4 is AGPLv3 licensed
+# DO NOT UPGRADE beyond ~> 7.5.7 without consulting legal, Grafana >= 8.0 is AGPLv3 Licensed
```
Comment (slimsag) [inline docker-images/grafana/Dockerfile:12]: What does ~> mean? I've never seen it

Comment (slimsag) [reply docker-images/grafana/Dockerfile:12]:
```suggestion
# DO NOT UPGRADE to AGPL Grafana without consulting Stephen+legal, Grafana >= 8.0 is AGPLv3 Licensed
```

File: docker-images/grafana/Dockerfile [hunk 2]
```diff
@@ -9,7 +9,7 @@ RUN ls '/generated/grafana'
...
 # See https://docs.google.com/document/d/1nSmz1ChL_rBvX8FAKTB-CNzgcff083sUlIpoXEz6FHE/edit?usp=sharing for details
```
Comment (slimsag) [inline docker-images/grafana/Dockerfile:13]:
```suggestion
# See https://docs.google.com/document/d/1nSmz1ChL_rBvX8FAKTB-CNzgcff083sUlIpoXEz6FHE/edit#heading=h.69clsrno4211
```

Linking to that new header / convo.

**Review by slimsag — APPROVED** [review]

File: docker-images/grafana/Dockerfile (hunk 1 above)
Comment (daxmc99) [reply docker-images/grafana/Dockerfile]: It is a [terraform constraint syntax](https://www.terraform.io/docs/language/expressions/version-constraints.html#gt--1), fine to remove it

Comment (slimsag) [reply docker-images/grafana/Dockerfile]: Interesting, thanks for the info!

**Pull request merged**